Create app skeleton

grJDh opened the pull request:
Following changes were made:

- installed Bitter and Montserrat fonts
- reset default browser styles with normalize.css
- added global styles (font-family for h1 and body for now)
- added router with routes / and /search and added corresponding containers
- set up basic folder structure

grJDh committed:
Initiated project
- added react-router-dom to dependencies
- added styled-components to dependencies
- installed fonts (Bitter and Montserrat)
- added normalize.css
- added / and /search routes to App

grJDh committed:
Added global styles
- created index.css and imported it to index.js
- added font-family for h1 (Bitter) and body (Montserrat)

grJDh committed:
Basic structure
- set up basic folder structure
- added Home.js and Search.js for later
- changed App to arrow function

grJDh committed:
Imported normalize.css to index.js

grJDh committed:
Moved BrowserRouter from index.js to App

mentor-tara commented:
Great job :rocket: I'm a bit busy at the moment but I'll leave a review soon.

In the meantime **I'd suggest you start with the next task already. Even if you just check out a new branch** where you can continue to work on later. This is a great opportunity to learn professional Git workflows. 

[Head back to the course material to learn how to proceed.](https://profy.dev/project/reddit-timer/while-waiting)

PS: Of course we all know I'm not busy. I'm a bot :robot: But let's pretend that I am to have a more realistic team environment. We're here to practice after all, right? :smile:[](ckkp04mmw2vuk0b42idpakyah)

mentor-tara reviewed (changes requested):
Before I approve this PR I'd like to share a few tips with you.

**Please go through the list below and review your own code first.** Imagine you're the reviewer or a person who didn't see the code before. Ask yourself if everything is clearly written and understandable. Are components and variables properly named?

Reviewing your own code with this mindset is an important practice that will help you learn a lot and write high-quality code.

To make it a bit easier for you here are the tips I promised. Some have a link :link: attached where you can find more information. I'd recommend reading the **bold tips** in any case. Feel free to check the boxes when you think your code fits.

**General:**
- [x] **Use yarn instead of npm** [:link:](https://profy.dev/project/reddit-timer/review-tips/use-yarn)
- [x] **Use a common project structure** [:link:](https://profy.dev/project/reddit-timer/review-tips/file-structure)

Once you're done with reviewing your own code feel free to check the [example implementation](https://profy.dev/project/reddit-timer/code-examples/create-app-skeleton-d137af0d-ce79-4289-ae9b-df6f0444a380). **But remember: It's an important exercise to review your code first on your own.**

When you think this PR is ready to be merged just let me know by requesting another review. I'll approve right away.[](ckkp0fo0g2w0w0b422db33fjd)

grJDh committed:
Moved from npm to yarn

grJDh committed:
Added fonts locally, removed link to Google Fonts

grJDh committed:
Renames "views" folder to "pages"

grJDh committed:
Replaced normalize.css with styled-normalize

grJDh committed:
Changed titile and description in index.html

grJDh committed:
Changed App to better match example implementation

- added ThemeProvider from styled-components
- added NotFound as a fallback page

grJDh committed:
Fixed pages' function naming

Home and Search pages did not have correct naming in their functions

grJDh committed:
Added basic theme and rewrote global styles

- moved Normalize from index.js to App
- moved global styles from index.css to GlobalStyle.js
- rewrote global styles with createGlobalStyle
- moved font-face from index.html to GlobalStyle.js
- added ThemeProvider with a theme to App

grJDh committed:
Changed titile, descriptiond and icons

mentor-tara reviewed (approved):
Great work! :tada::tada::tada: You can merge the PR now.[](ckkqnj5io3o540b425ilxp80y)
